packages/wonder-blocks-dropdown/components/multi-select.md

```diff
@@ -316,4 +316,30 @@ const {LabelLarge} = require("@khanacademy/wonder-blocks-typography");
         <OptionItem label="some value" value="" />
     </MultiSelect>
 </View>
-```
\ No newline at end of file
+```
+
+### Implicit all enabled Multi select
+
```

Comment:
Could you please add the `onChange` prop to this example to see how the logic would work with `implicitAllEnabled`? (something similar to the previous MultiSelect examples)